app/portainer/views/endpoints/create/createendpoint.html

```diff
@@ -36,6 +36,16 @@
                   <p>Portainer agent</p>
                 </label>
               </div>
+              <div ng-click="setDefaultPortainerInstanceURL()">
+                <input type="radio" id="edge_agent_endpoint" ng-model="state.EnvironmentType" value="edge_agent">
+                <label for="edge_agent_endpoint">
+                  <div class="boxselector_header">
+                    <i class="fa fa-cloud" aria-hidden="true" style="margin-right: 2px;"></i>
+                    Edge Agent
+                  </div>
+                  <p>Portainer Edge agent</p>
+                </label>
+              </div>
               <div>
```

Comment:
As there is 2 resets of URL above for other choices than Edge Endpoints, shouldn't the URL be reset here too ?

Reply:
It's using `setDefaultPortainerInstanceURL()` to set the default value of the URL when using an Edge agent endpoint.

Reply:
I was targeting the following code

```
<div>
  <input type="radio" id="azure_endpoint" ng-model="state.EnvironmentType" value="azure">
  <label for="azure_endpoint">
    <div class="boxselector_header">
      <i class="fab fa-microsoft" aria-hidden="true" style="margin-right: 2px;"></i>
        Azure
    </div>
    <p>Connect to Microsoft Azure ACI</p>
  </label>
</div>
```
To change to
```
<div ng-click="resetEndpointURL()">
  <input type="radio" id="azure_endpoint" ng-model="state.EnvironmentType" value="azure">
  <label for="azure_endpoint">
    <div class="boxselector_header">
      <i class="fab fa-microsoft" aria-hidden="true" style="margin-right: 2px;"></i>
        Azure
    </div>
    <p>Connect to Microsoft Azure ACI</p>
  </label>
</div>
```

As you did an URL reset for docker and agent endpoints

Reply:
Doesn't really matter as the endpoint URL is ignored when using an Azure endpoint.

Reply:
Good point. All good then !